types/redux-first-router/index.d.ts

```diff
@@ -80,25 +99,29 @@ export interface NavigationAction {
     routeName?: string;
     actions?: NavigationAction[];
     action?: NavigationAction;
-    params?: object;
-    meta?: object;
+    params?: Params;
+    meta?: Meta;
```

Comment:
might have been a bad find-replace...I'll go back over this tonight and make sure the Flow types and this PR match exactly.